Security.md

```diff
@@ -0,0 +1,122 @@
+# Security
+
+The security model of WebAssembly has two important goals: (1) protect *users*
+from buggy or malicious modules, and (2) provide *developers* with useful
+primitives and mitigations for developing safe applications, within the
+constraints of (1).
+
+## Users
+
+Each WebAssembly module executes within a sandboxed environment that is isolated
+from the host runtime using software-based fault isolation. This implies:
```

Comment:
That the fault isolation is software-based in most of today's implementations is an implementation detail; would it still be clear to say "using fault isolation techniques"?